ee/src/main/java/org/jboss/as/ee/logging/EeLogger.java

```diff
@@ -206,17 +206,21 @@ public interface EeLogger extends BasicLogger {
     @Message(id = 14, value = "%s in subdeployment ignored. jboss-ejb-client.xml is only parsed for top level deployments.")
     void subdeploymentIgnored(String pathName);
 
-    //@Message(id = 15, value = "Transaction started in EE Concurrent invocation left open, starting rollback to prevent leak.")
-    //void rollbackOfTransactionStartedInEEConcurrentInvocation();
+    @LogMessage(level = WARN)
+    @Message(id = 15, value = "Transaction started in EE Concurrent invocation left open, starting rollback to prevent leak.")
+    void rollbackOfTransactionStartedInEEConcurrentInvocation();
 
-    //@Message(id = 16, value = "Failed to rollback transaction.")
-    //void failedToRollbackTransaction(@Cause Throwable cause);
+    @LogMessage(level = WARN)
+    @Message(id = 16, value = "Failed to rollback transaction.")
+    void failedToRollbackTransaction(@Cause Throwable cause);
 
-    //@Message(id = 17, value = "Failed to suspend transaction.")
-    //void failedToSuspendTransaction(@Cause Throwable cause);
+    @LogMessage(level = WARN)
+    @Message(id = 17, value = "Failed to suspend transaction.")
+    void failedToSuspendTransaction(@Cause Throwable cause);
 
-    //@Message(id = 18, value = "System error while checking for transaction leak in EE Concurrent invocation.")
-    //void systemErrorWhileCheckingForTransactionLeak(@Cause Throwable cause);
+    @LogMessage(level = WARN)
+    @Message(id = 18, value = "System error while checking for transaction leak in EE Concurrent invocation.")
+    void systemErrorWhileCheckingForTransactionLeak(@Cause Throwable cause);
```

Comment:
I'm not sure on the solution here, but the logged ID will be different which we've tried to avoid in the past. This is just more of an FYI than anything.

Reply:
Those messages were created and used by the factory we are moving back to EE subsystem so I assume we can reactive these again?

Reply:
👍

Reply:
Yes, they can definitely be enabled. This was just more of a general comment as the idea of the ID is to not change them so they can be tracked to the message. That said with the Wildscribe sutff I guess this is less of an issue.